### DISCUSSION TITLE
Version 1.42.1 (2023-11-07)

### DISCUSSION THREAD

**Discussion by dnwe** [IBM/sarama#2712] [Announcements]
### :bug: Fixes
* fix: make fetchInitialOffset use correct protocol by @dnwe in https://github.com/IBM/sarama/pull/2705
* fix(config): relax ClientID validation after 1.0.0 by @dnwe in https://github.com/IBM/sarama/pull/2706

**Full Changelog**: https://github.com/IBM/sarama/compare/v1.42.0...v1.42.1

<hr /><em>This discussion was created from the release <a href='https://github.com/IBM/sarama/releases/tag/v1.42.1'>Version 1.42.1 (2023-11-07)</a>.</em>

**Comment by JeffBNimble** [comment]
Is anyone having trouble with the latest v1.42.1 release? I pulled it in and now my service panics with either a "broken pipe" error or an EOF error when attempting to consume. I reverted back to 1.42.0 and everything works fine.

**Reply by dnwe** [reply]
@JeffBNimble hmm, please can you open an issue? What's your remote Kafka cluster version and/or provider (AWS / Azure / IBM / Confluent)?

**Reply by dnwe** [reply]
Note: this was raised as https://github.com/IBM/sarama/issues/2719 and eventually closed out as being config-related
